src/main/java/com/matera/cadastrocentral/address/Address.java

```diff
@@ -0,0 +1,103 @@
+package com.matera.cadastrocentral.address;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.PrePersist;
+import java.util.UUID;
+
+@Entity
+public class Address {
+
+    private @Id UUID addressId;
+    private UUID clientId;
+    private String streetName;
+    private String district;
+    private String state;
+    private String zipCode;
+    private String complement;
+    private String reference;
+    private String city;
+
+
+    @PrePersist
+    public void prePersist() {
+        this.addressId = UUID.randomUUID();
+    }
+
+    public Address(UUID clientId, String streetName, String district, String state, String zipCode, String complement, String reference) {
```

Comment:
looks like the constructor is not used anywere, can you remove it?

Reply:
How will the application know how to create the instance if I delete the constructor? or does Spring automaticaly figure it out?